src/storage/backup/Cargo.toml

```diff
@@ -16,12 +16,14 @@ normal = ["workspace-hack"]
 [dependencies]
 anyhow = "1"
 async-trait = "0.1"
+bincode = "1.3"
 bytes = { version = "1", features = ["serde"] }
 itertools = "0.11"
 parking_lot = { version = "0.12", features = ["arc_lock"] }
 prost = { workspace = true }
 risingwave_common = { workspace = true }
 risingwave_hummock_sdk = { workspace = true }
+risingwave_meta_model_v2 = { workspace = true }
```

Comment:
Just curious: why there were no related dependencies for v1? :eyes: 
(Would this introduce any concern for slow compilation?)

Reply:
Because v1 uses structs generated by prost directly, while v2 uses structs defined as seaorm model.